include/rnp/rnp_def.h

```diff
@@ -88,6 +89,11 @@ enum {
     RNP_ERROR_MAC_INVALID,
     RNP_ERROR_SIGNATURE_INVALID,
     RNP_ERROR_KEY_GENERATION,
+    RNP_ERROR_KEY_NOT_FOUND,
+    RNP_ERROR_NO_SUITABLE_KEY_FOR_OPERATION,
```

Comment:
(Not important but `RNP_ERROR_NO_SUITABLE_KEY` seems descriptive enough)

Reply:
Better, thanks